src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1605,8 +1605,8 @@ static void perform_stream_op(grpc_transport* gt, grpc_stream* gs,
 }
 
 static void cancel_pings(grpc_chttp2_transport* t, grpc_error_handle error) {
-  GRPC_CHTTP2_IF_TRACING(
-      gpr_log(GPR_INFO, "%p CANCEL PINGS: %s", t, error.ToString().c_str()));
+  GRPC_CHTTP2_IF_TRACING(gpr_log(GPR_INFO, "%p CANCEL PINGS: %s", t,
+                                 grpc_core::StatusToString(error).c_str()));
```

Comment:
For my education: why is ToString not the right thing?

Reply:
On Windows at least, the child errors are printed with embedded hex when you use status.ToString(). Try to pick out the file line number for the child error in this example: 

```
D0620 15:33:35.497000000 19492 src/core/ext/transport/chttp2/transport/chttp2_transport.cc:2198] MARK_STREAM_CLOSED: t=000001C9618D5830 s=000001C9618DB710(id=1) read+write [UNKNOWN: Endpoint read failed [type.googleapis.com/grpc.status.str.file='src/core/ext/transport/chttp2/transport/chttp2_transport.cc'] [type.googleapis.com/grpc.status.int.file_line='2534'] [type.googleapis.com/grpc.status.time.created_time='2023-06-20T22:33:35.497401355+00:00'] [type.googleapis.com/grpc.status.children='\'\x01\x00\x00\x08\x02\x12\x44\x41n existing connection was forcibly closed by the remote host.%0D%0A\x1aX\n1type.googleapis.com/grpc.status.time.created_time\x12#2023-06-20T22:33:35.497371459+00:00\x1a\x34\n-type.googleapis.com/grpc.status.int.file_line\x12\x03\x31\x39\x30\x1aM\n(type.googleapis.com/grpc.status.str.file\x12!src/core/lib/iomgr/tcp_windows.cc'] [type.googleapis.com/grpc.status.int.occurred_during_write='0']]
```